cli/command/service/ps.go

```diff
@@ -58,8 +59,11 @@ func runPS(dockerCli *command.DockerCli, opts psOptions) error {
 	serviceIDFilter := filters.NewArgs()
 	serviceNameFilter := filters.NewArgs()
 	for _, service := range opts.services {
+		// default to container runtime
 		serviceIDFilter.Add("id", service)
+		serviceIDFilter.Add("runtime", string(swarmtypes.RuntimeContainer))
```

Comment:
IIUC, this adds a new filter option to `docker service ls`, and the API for that command? If so, please add the new filter to the CLI docs; https://github.com/docker/docker/blob/master/docs/reference/commandline/service_ps.md#filtering, and the API history https://github.com/docker/docker/blob/master/docs/api/version-history.md#v129-api-changes